test/lru_cache_test.rb

```diff
@@ -0,0 +1,63 @@
+require 'test_helper'
+
+class TestLRUCache < Minitest::Test
+  def setup
+    @lru_cache = LRUCache.new(3)
```

Comment:
I'm pretty sure an LRU of size 3 isn't big enough to test all the branches. Maybe size 5?